tools/scenario_runner.py

```diff
@@ -256,8 +254,6 @@ def run(
                 if our_index == 0:
                     last_node = nodes[-1]
                     transfers_by_peer[last_node] = int(amount)
-            else:
```

Comment:
Why is this removed?

Reply:
`peer` is not used in the else branch or in the code after the `if/else`.